src/base/constants.py

```diff
@@ -43,4 +42,9 @@ def _IsDevAppServer():
 
 DEBUG = IS_DEV_APPSERVER
 
-TEMPLATE_DIR = os.path.sep.join([os.path.dirname(__file__), '..', 'templates'])
+template_dir = ('static' if os.environ.get('AUTHORIZATION_METHOD')
```

Comment:
Sorry, my eyes bleed when I read this. Have you heard of principle of least surprise? I would be very surprised (and very upset) if a random unrelated setting was changed when toggling the authentication method.

This either needs 1. a short paragraph of very careful and detailed explanation for why it's done this way, OR 2. it must be so simple/obvious that it doesn't need an explanation.

> It's done because when we are rendering static files, mostly they are in static/ so thats why. This is used in base code and integrated with jinja2 configs.

Sorry but this looks like adding unnecessary complexity / indirection.